src/wiki/ventoy.md

```diff
@@ -0,0 +1,106 @@
+6. Click install, and Ventoy will create a bootable drive for you. After it finishes, you will see a drive named as "Ventoy", this is where you will place your ISO files for the bootable drive.
+3. For this time, we will use the Ventoy2Disk.exe, we click on it, and we click to “yes” in the UAC screen that pooped.
+4. Then at devices section select your USB that you want to make a Ventoy or if you can't see it press the green button to refresh the list.
+## Downloading Windows to the Bootable Drive
+### Downloading the ISO file
+![Menu for selecting settings in Ventoy](<img/ventoy/Secure boot and GPT.png>)
+6. At the last step, click on install and wait some seconds to get the job done.
+For Windows 11, go to [https://www.microsoft.com/software-download/windows11](https://www.microsoft.com/software-download/windows11)
+::: tip
+If you are not on Windows, you will see an option to "Select Edition". Select the proper edition, and language, and download the file. Once the download finishes, move the ISO file to the Ventoy drive.
+:::
+
+## Downloading windows
+### Downloading the tool for downloading the ISO
+
+1. For Windows 10, go to [https://www.microsoft.com/software-download/windows10](https://www.microsoft.com/software-download/windows10) <br>
+   For Windows 11, go to [https://www.microsoft.com/software-download/windows11](https://www.microsoft.com/software-download/windows11)
+
+2. Click the **Download Tool Now** button.
+
+   ![](./img/downloading-windows/downloadtoolnow.png)
+2. Wait for the installer to prepare the files needed.
+3. After the tool has been downloaded, run it.
+
+### Using the tool
+
+1. The first thing you will see is this screen, click Accept.
+Select **Create installation media for another PC**, then press next.
+   ![](./img/downloading-windows/licenseterms.png)
+
+2. The next screen you see will be this, wait for the installer to complete preparation.
+4. Use the recommended settings, or select your preferred Language, Edition, and Architecture.
+   ![](./img/downloading-windows/gettingthings.png)
+::: tip Note
+If you're planning to install Windows on another PC and don't know what architecture it uses, then select "Both".
+:::
+   — You need to click on:**want to reinstall Windows on another computer** click **Create installation media**.
+
+    ![](./img/downloading-windows/whatdoyouwanttodo.png)
+5. Select **ISO file**, then press next.
+4. Select the correct Architecture, Edition, and Language of Windows, or use the recommended settings.
+
+   ::: tip Note
+   If you're planning on installing Windows on another PC and don't know what to select for the architecture, then choose both x86 and x64.
+   :::
+
+   ![](./img/downloading-windows/changelanguagearchitecture.png)
+Once the file has been downloaded, copy it to the Ventoy Drive.
+5. On the Media Creation Tool, select **ISO file**.
+
+You can read more about reinstalling Windows [here](installing-windows).
+
+6. Select a path to save to.
```

Comment:
Didn't we literally just download windows earlier? Perhaps make this a 2-method process under Downloading Windows, and 1 method would be the website and the other would be the MCT.